cmd/snapd/main.go

```diff
@@ -58,13 +59,20 @@ func main() {
 	ch := make(chan os.Signal, 2)
 	signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM)
 	if err := run(ch); err != nil {
-		if err == daemon.ErrRestartSocket {
+		if errors.Is(err, daemon.ErrRestartSocket) {
 			// Note that we don't prepend: "error: " here because
 			// ErrRestartSocket is not an error as such.
 			fmt.Fprintf(os.Stdout, "%v\n", err)
 			// the exit code must be in sync with
 			// data/systemd/snapd.service.in:SuccessExitStatus=
 			os.Exit(42)
+		} else if errors.Is(err, daemon.ErrNoFailureRecoveryNeeded) {
+			// Similar consideration as above.
+			fmt.Fprintf(os.Stdout, "%v\n", err)
```

Comment:
Isn't this the same as
```suggestion
			fmt.Println(err)
```

Reply:
replaced with `fmt.Fprintln(os.Stdout..)` here and above for consistency with other code using `fmt.F*` functions.